source/languages/en/riak/dev/taste-of-riak/nodejs.md

```diff
@@ -0,0 +1,176 @@
+
+var storeFuncs = [];
+people.forEach(function (person) {
+    // Create functions to execute in parallel to store people
+    storeFuncs.push(function (async_cb) {
+        client.storeValue({
+                bucket: 'contributors',
+                key: person.emailAddress,
+                value: person
+            },
+            function(err, rslt) {
+                async_cb(err, rslt);
+            }
+        );
+    });
+};
+
+async.parallel(storeFuncs, function (err, rslts) {
+    if (err) {
+        throw new Error(err);
+    }
+});
+```
+
+In this sample, we create a collection of `Person` objects and then save each `Person` to Riak. Once again, we check the response from Riak.
+
+### Reading from Riak
+
+Let's find a person!
+
+```javascript
+var logger = require('winston');
+
+client.fetchValue({ bucket: 'contributors', key: 'bashoman@basho.com', convertToJs: true },
+    function (err, rslt) {
+        if (err) {
+            throw new Error(err);
+        } else {
+            var riakObj = rslt.values.shift();
+            var bashoman = riakObj.value;
+            logger.info("I found %s in 'contributors'", bashoman.emailAddress);
+        }
+    }
+);
+```
+
+We use `client.fetchValue` to retrieve an object from Riak. This returns an array of `RiakObject` objects which helpfully encapsulates the communication with Riak.
+
+After verifying that we've been able to communicate with Riak *and* that we have a successful result, we use the `value` property to get the object, which has already been converted to a javascript object due to the use of `convertToJs: true` in the options.
+
+### Modifying Existing Data
+
+Let's say that Basho Man has decided to be known as Riak Man:
+
+```javascript
+bashoman.FirstName = "Riak";
+riakObj.setValue(bashoman);
+
+client.storeValue({ value: riakObj }, function (err, rslt) {
+    if (err) {
+        throw new Error(err);
+    }
+});
+```
+
+Updating an object involves modifying a `RiakObject` then using `client.storeValue` to save the existing object. 
+
+### Deleting Data
+
+```javascript
+client.deleteValue({ bucket: 'contributors', key: 'johndoe@gmail.com' }, function (err, rslt) {
+    if (err) {
+        throw new Error(err);
+    }
+};
+```
+
+Just like other operations, we check the results that have come back from Riak to make sure the object was successfully deleted. Of course, if you don't care about that, you can just ignore the result.
+
+The Riak NodeJS Client has a lot of additional functionality that makes it easy to build rich, complex applications with Riak. Check out the [documentation](https://github.com/basho/riak-dotnet-client/wiki) to learn more about working with the Riak NodeJS Client and Riak.
+
+## Next Steps
+
+More complex use cases can be composed from these initial create, read, update, and delete (CRUD) operations. [[In the next chapter|Taste of Riak: Querying]], we will look at how to store and query more complicated and interconnected data, such as documents.
```

Comment:
It looks like the `title` for the linked page is `Taste of Riak: Querying with NodeJS`. It would probably be worth testing out all the links before merging these changes.